sympy/utilities/lambdify.py

```diff
@@ -386,6 +386,14 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     namespace.update({'builtins':builtins, 'range':range})
 
     func = eval(lstr, namespace)
+    # For numpy lambdify, wrap all input arguments in arrays.
+    # This is a fix for gh-11306.
+    if module_provided and _module_present('numpy',namespaces):
```

Comment:
No, we don't enforce PEP 8 (only trailing whitespace, and some other issues), but we do generally try to follow it, so there should be a space after commas.